app/app/urls.py

```diff
@@ -514,6 +514,21 @@
         retail.emails.successful_contribution,
         name='admin_successful_contribution'
     ),
+    path(
+        '_administration/email/hackathon_ends/<str:hackathon>/<str:username>',
+        retail.emails.hackathon_ends,
+        name='hackathon_ends'
+    ),
+    path(
+        '_administration/email/bounty_not_submitted/<int:bounty>',
+        retail.emails.bounty_not_submitted,
+        name='bounty_not_submitted'
+    ),
+    path(
+        '_administration/email/bounty_added_to_event/<int:bounty>/<str:profile>',
```

Comment:
^ could we update URLs to `_administration/email/hackathons/*`
All the URLs

The other we'll have to group off them slowly to make it a li more maintainable

Reply:
Done.